tests/mocks/sip.py

```diff
@@ -0,0 +1,45 @@
+from palace.manager.api.sip import SIPClient
+from palace.manager.api.sip.client import Constants
+
+
+class MockSIPClient(SIPClient):
```

Comment:
This isn't new, but it was in the main package, so it got moved out to `mocks` as part of this re-org.